tests/libtransmission/torrent-files-test.cc

```diff
@@ -144,23 +144,45 @@ TEST_F(TorrentFilesTest, hasAnyLocalData)
 TEST_F(TorrentFilesTest, isSubpathPortable)
 {
     static auto constexpr Tests = std::array<std::pair<std::string_view, bool>, 15>{ {
```

Comment:
nit: Maybe it would be easier to define
```c
TEST_F(TorrentFilesTest, isSubpathPortable)
{
#ifdef __APPLE__
#define RESERVED_NOT_APPLE true
#else
#define RESERVED_NOT_APPLE false
#endif
    static auto constexpr Tests = std::array<std::pair<std::string_view, bool>, 15>{ {
        // don't end with periods
        { "foo.", RESERVED_NOT_APPLE },
        { "foo..", RESERVED_NOT_APPLE },
        ...
    } };
#undef __APPLE__

```
and just use that instead of `true`/`false` here.

Reply:
I figured out that all the Apple rules are identical to Linux rules.
So yes, I can do a RESERVED_WIN32.

Reply:
done in a8a4774